core/vote/vote_pool.go

```diff
@@ -23,6 +24,9 @@ const (
 	upperLimitOfVoteBlockNumber = 11 // refer to fetcher.maxUncleDist
 
 	chainHeadChanSize = 10 // chainHeadChanSize is the size of channel listening to ChainHeadEvent.
+
+	fetchCheckFrequency = 100 * time.Millisecond
```

Comment:
I think 100 ms is too much, it should be smaller. Its turn can be taken over by other locks if the waiting time is too long. In my opinion, we should set it small as 1ms and increase the fetchRetry number

Reply:
I've changed these constants, my only concern here is if the waiting time is too short, when constantly trying to acquire the lock, we make the cache bouncing between CPUs which degrades the system performance.